src/test/java/seedu/address/model/note/TagNoteMapTest.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.model.note;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.TypicalClients.ALICE;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.client.Client;
+import seedu.address.model.tag.Tag;
+
+class TagNoteMapTest {
+
+    private static final Tag TAG = new Tag("tagName");
+    private Client client = ALICE;
+    private final Note taggedNote = new Note("jurong hill was a nice place");
+    private final TagNoteMap tagNoteMap = new TagNoteMap();
+
+    @Test
+    void initMapFromClients_addClientWithTaggedNotes_returnsTrue() {
```

Comment:
```suggestion
    void initMapFromClients_addClientWithTaggedNotes_success() {
```

This method doesn't seem to return a boolean, so `success` should be used instead. Or you could change it to `doesNotThrowException`. The other 2 methods below also need their method names updated, `returnsTrue` should be `<expected outcome>`.